public/editor/scripts/main.js

```diff
@@ -89,6 +89,7 @@ require(["jquery", "bowser", "analytics"], function($, bowser, analytics) {
     var appUrl = thimbleScript.getAttribute("data-app-url");
     var projectDetails = thimbleScript.getAttribute("data-project-details");
     var editorUrl = thimbleScript.getAttribute("data-editor-url");
+    clearTimeout(slowLoadingTimeout);
```

Comment:
Yeah, that's pretty much why I put it here, just to keep all the related functionality in one place. You're right about ``bramble-ui-bridge`` though. Seems like it belongs in there and was also my initial, pre-lazy assessment.